Editor/Resources/EditorWindow/Components/RegisterComputeInput.uxml

```diff
@@ -0,0 +1,35 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <ui:Style src="./common.uss"/>
+    <ui:VisualElement class="separator">
+        <ui:VisualElement class="form-row">
+            <ui:Label name="AnywherePageComputeNameLabel" class="form-row__label" text="Compute Name"/>
+            <ui:TextField name="AnywherePageComputeNameInput" class="form-row__input text-input" value="ComputerName-ProfileName"/>
+        </ui:VisualElement>
+        <ui:VisualElement class="form-row">
+            <ui:Label name="AnywherePageComputeIPLabel" class="form-row__label" text="IP Address"/>
+            <ui:VisualElement class="form-row__input form-row__input--inline">
+                <ui:TextField name="AnywherePageComputeIPAddressInput" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--fitted text-input--centered"/>
+                <ui:Label text="." class="ip-spacer"/>
+                <ui:TextField name="AnywherePageComputeIPAddressInput" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--fitted text-input--centered"/>
+                <ui:Label text="." class="ip-spacer"/>
+                <ui:TextField name="AnywherePageComputeIPAddressInput" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--fitted text-input--centered"/>
+                <ui:Label text="." class="ip-spacer"/>
+                <ui:TextField name="AnywherePageComputeIPAddressInput" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--fitted text-input--centered"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement name="AnywherePageComputeStatus" class="form-row">
+            <ui:Label name="AnywherePageComputeStatusLabel" class="form-row__label" text="Status"/>
+            <ui:Label name="AnywherePageComputeStatusDisplay" class="form-row__text-display" text="Registered"/>
+        </ui:VisualElement>
+        <ui:VisualElement class="form-row">
+            <ui:VisualElement class="form-row__input form-row__input--no-label separator separator--horizontal">
+                <ui:Button name="AnywherePageComputeRegisterButton" class="button button--primary" text="Register Compute"/>
+                <ui:Button name="AnywherePageComputeReplaceComputeButton" class="button" text="Register new Compute"/>
+                <ui:Button name="AnywherePageComputeCancelReplace" class="button" text="Cancel Compute"/>
```

Comment:
minor: This should be `AnywherePageComputeCancelReplaceButton` instead.

Reply:
renamed